packages/config/src/layer2s/types/Layer2Url.ts

```diff
@@ -0,0 +1,16 @@
+interface AlchemyProvider {
+  provider: 'alchemy'
+  slug: string
+}
+
+interface JsonRpcProvider {
+  provider: 'jsonRpc'
+  url: string
+}
+
+export type Layer2Rpc = (AlchemyProvider | JsonRpcProvider) & {
+  type: 'rpc'
+  callsPerMinute?: number
+}
+
+export type Layer2Url = Layer2Rpc
```

Comment:
How about a different name, more suggestive of the actual use, like: `Layer2TransactionApi`